assemblies/assembly-planner-configuration.adoc

```diff
@@ -554,7 +554,9 @@ To configure your OptaPlanner Quarkus application to use Micrometer and a specif
 * You have a Quarkus OptaPlanner application.
 
 .Procedure
-. Add the following dependency to your application's `pom.xml` file where `<MONITORING_SYSTEM>` is a monitoring system supported by Micrometer:
+. Add the following dependency to your application's `pom.xml` file where `<MONITORING_SYSTEM>` is a monitoring system supported by Micrometer and Quarkus:
++
+NOTE: Prometheus is currently the only monitoring system supported by Quarkus.
```

Comment:
Officially supported; the rest of the monitoring systems are community supported in the Quarkiverse; the rest of the registries either need to be manually set up (like plain Java) or use a community extension (https://github.com/quarkiverse/quarkus-micrometer-registry)